bucket/tribler.json

```
@@ -0,0 +1,44 @@
+{
+	"version": "7.2.2",
+	"description": "Privacy enhanced BitTorrent client with P2P content discovery",
+	"homepage": "http://www.tribler.org",
+	"license": "LGPL-3.0",
+	"architecture": {
+		"64bit": {
+			"url": "https://github.com/Tribler/tribler/releases/download/v7.2.2/Tribler_7.2.2_x64.exe#/dl.7z",
+			"hash": "794ecbae6578e285f73def12c5504faa5411088903b24d9b02faad58f5570456"
+		},
+		"32bit": {
+			"url": "https://github.com/Tribler/tribler/releases/download/v7.2.2/Tribler_7.2.2_x86.exe#/dl.7z",
+			"hash": "9684de83d64b00b4a386c3dcebbb2324b8f463a7ffbd7afd9ada19bb8858b9e0"
+		}
+	},
+	"bin": "tribler.exe",
+	"shortcuts": [
+		[
+			"tribler.exe",
+			"Tribler"
+		]
+	],
+	"checkver": {
+		"github": "https://github.com/Tribler/tribler"
+	},
+	"autoupdate": {
+		"architecture": {
+			"64bit": {
+				"url": "https://github.com/Tribler/tribler/releases/download/v$version/Tribler_$version_x64.exe#/dl.7z",
+				"hash": {
+					"url": "$baseurl/SHA256.txt",
+					"regex": "$sha256  Tribler_$version_x64.exe"
+				}
+			},
+			"32bit": {
+				"url": "https://github.com/Tribler/tribler/releases/download/v$version/Tribler_$version_x86.exe#/dl.7z",
+				"hash": {
+					"url": "$baseurl/SHA256.txt",
+					"regex": "$sha256  Tribler_$version_x86.exe"
+				}
+			}
+		}
```

use generic hash. It is proper digest format, so should be enough with just this:

```suggestion
		},
		"hash": {
		    "url": "$baseurl/SHA256.txt"
		}
```